docs/Reference/CheckBox.md

```diff
@@ -825,6 +825,7 @@ export default class Plugin extends React.Component {
     }
 
     render() {
+        return
```

Comment:
Syntax error:
return
         <div>
           <CheckBox items={this.state.items} value={this.state.value} onChange={(value) => {this.setState({value})}} />
           <button onClick={this.handleClick}>Click</button>
         </div>
Need change to:
return (
         <div>
           <CheckBox items={this.state.items} value={this.state.value} onChange={(value) => {this.setState({value})}} />
           <button onClick={this.handleClick}>Click</button>
         </div>
);